### PR TITLE
Improve function signatures in aqua command

### PR DESCRIPTION
**Pull request by folex**
- Show argument names in function signatures: `add_service_id(node_id: string, service_id: string)`
- Show empty parens if no arguments: `get_nodes() -> NodesResponse`
- Service name is now CamelCase
- New parameter `--id`/`-i` to specify service id: `--id dht --service dht_service` => `DhtService("dht"):`
- Remove extra newline between function signatures

Example output:
```
service DhtService("dht"):
  update(peer_id: string, relay_id: string, nickname: string) -> EmptyResponse
  add_service_id(node_id: string, service_id: string) -> EmptyResponse
  get_nodes() -> NodesResponse
  new_message(msg: Message) -> EmptyResponse
```

### CODE REVIEW

File: tools/cli/src/uppercase.rs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#[cfg(test)]
+mod tests {
+    use crate::uppercase::uppercase_first_letter;
+
+    #[test]
+    fn uppercase() {
+        let s = String::from("hello, world!");
+        let result = uppercase_first_letter(&s);
+        assert_eq!(s, "Hello, world!");
```
Comment (mikevoronov) [inline tools/cli/src/uppercase.rs:34]: Are you shure about `s` here?

File: tools/cli/src/uppercase.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    #[test]
+    fn identity() {
+        let s = String::from("Hello, world!");
+        let result = uppercase_first_letter(&s);
+        assert_eq!(s, "Hello, world!");
```
Comment (mikevoronov) [inline tools/cli/src/uppercase.rs:41]: and here

File: tools/cli/src/main.rs [hunk 3]
```diff
@@ -89,20 +91,24 @@ fn aqua(args: &clap::ArgMatches<'_>) -> Result<(), anyhow::Error> {
         println!("{}", record);
     }
 
-    match args.value_of(args::SERVICE_NAME) {
-        Some(service_name) => println!("service {}:", service_name.to_title_case()),
```
Comment (mikevoronov) [inline tools/cli/src/main.rs:93]: What was bad with `.to_title_case()`?

Comment (mikevoronov) [reply tools/cli/src/main.rs:93]: I got it, but maybe it's better to find a crate for that?

Comment (mikevoronov) [reply tools/cli/src/main.rs:93]: Or, probably, [to_class_case](https://docs.rs/Inflector/0.11.4/inflector/cases/classcase/index.html) is what we need here

File: tools/cli/src/uppercase.rs (hunk 1 above)
Comment (folex) [reply tools/cli/src/uppercase.rs:34]: no, typo

File: tools/cli/src/main.rs (hunk 3 above)
Comment (folex) [reply tools/cli/src/main.rs:93]: sounds like a solution!

**Review by mikevoronov — APPROVED** [review]

**Pull request merged**